packages/react-scripts/config/webpack.config.js

```diff
@@ -340,21 +340,12 @@ module.exports = function(webpackEnv) {
                 // @remove-on-eject-begin
                 baseConfig: (() => {
                   const eslintCli = new eslint.CLIEngine();
-                  let eslintConfig;
-                  try {
-                    eslintConfig = eslintCli.getConfigForFile(paths.appIndexJs);
-                  } catch (e) {
-                    // A config couldn't be found.
-                  }
+                  const eslintConfig = eslintCli.getConfigForFile(
```

Comment:
This threw an error previously. I understand it doesn't now?

Reply:
I think I may have been getting lucky (eslint config somewhere up the file system tree perhaps?). I was seeing errors in CI so i've reverted to include the catch again.

I also observed a strange error on our PNP test that failed due to `eslint-config-react-app` not being defined in `package.json`. This catch block is catching that error on `master` it appears.